avo-hq/avo
# feature: find record method

# Description


Fixes https://github.com/avo-hq/avo/issues/1165

[Prerelease](https://rubygems.org/gems/avo/versions/2.25.1.pre.1.pr1579)
[Docs](https://docs.avohq.io/2.0/customization.html#custom-find-method-for-show-and-edit-pages)

# Checklist:




## Manual review steps


1. Go to `/admin/people`
2. select a person
3. observe the param is the person name, not the `id`

Manual reviewer: please leave a comment with output from the test if that's the case.
